python/python/glide/async_commands/cluster_commands.py

```diff
@@ -340,3 +341,88 @@ async def time(self, route: Optional[Route] = None) -> TClusterResponse[List[str
             TClusterResponse[List[str]],
             await self._execute_command(RequestType.Time, [], route),
         )
+
+    async def sort(
+        self,
+        key: str,
+        limit: Optional[Limit] = None,
+        order: Optional[OrderBy] = None,
+        alpha: Optional[bool] = None,
+    ) -> List[str]:
+        """
+        Sorts the elements in the list, set, or sorted set at `key` and returns the result.
+        To store the result into a new key, see `sort_store`.
+
+        By default, sorting is numeric, and elements are compared by their value interpreted as double precision floating point numbers.
+
+        See https://valkey.io/commands/sort for more details.
+
+        Args:
+            key (str): The key of the list, set, or sorted set to be sorted.
+            limit (Optional[Limit]): The limit argument for a range query. Defaults to None. See `Limit` class for more information.
```

Comment:
I would put `See `Limit` class for more information.` at the end. 
and change to `Limiting the range of the query by setting offset and result count.`
Then get into a bit more technical details. But its too long and too complicated to understand.

Reply:
`The limit argument for a range query. `is true for zrange, not here

Reply:
I believe the explanation about offset and count, and what to do if offset exceeds the range is necessary, this isnt the first command with this details